.github/workflows/test-changed-firestore.yml

```diff
@@ -3,8 +3,42 @@ name: Test Firestore
 on: pull_request
 
 jobs:
+  compat-test-chrome:
+    name: Test Firestore Compatible
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@master
+        with:
+          # This makes Actions fetch all Git history so run-changed script can diff properly.
+          fetch-depth: 0
+      - name: Set up Node (14)
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+      - name: install Chrome stable
+        run: |
+          sudo apt-get update
+          sudo apt-get install google-chrome-stable
+      - name: Bump Node memory limit
+        run: echo "NODE_OPTIONS=--max_old_space_size=4096" >> $GITHUB_ENV
+      - name: Test setup and yarn install
+        run: |
+          cp config/ci.config.json config/project.json
+          yarn
+      - name: build
```

Comment:
Instead of each of these jobs building the repo, can you create a job that builds once and then uploads the build artifact? That way the test jobs simply download the build artifact and run the tests.

It helps when there are flaky tests where you don't have to wait for a full build again when running a second attempt.

I recently did this [here in the Test All workflow](https://github.com/firebase/firebase-js-sdk/blob/c9e2b0b8cd5fd0db3cac7bc3a00629ae34302189/.github/workflows/test-all.yml#L14).

Reply:
Done.